---
layout: post
title: Wednesday Update
date: '2023-04-12'
categories: daily_log geoduck results
tags: update index 
---



# Blasting against P.gen with other transctiptomes
- Merceneria Mercenaria [Mercenaria stand alone blast](https://gannet.fish.washington.edu/gigas/data/p.generosa/Mercenaria_blastx.tab)
- Manilla Clam [Ruditapes philippinarum stand alone blast: job 4546808](https://gannet.fish.washington.edu/gigas/data/p.generosa/Pgen_Mercenaria_blastx.tab)
- Merceneria Against Pgenerosa transcriptome [Pgen_db_Mercenaria](https://gannet.fish.washington.edu/gigas/data/p.generosa/Mercenaria_Pgenenerosa_blastx.tab)
- Manilla Against Pgenerosa transcriptome [Pgen_db_Manilla](https://gannet.fish.washington.edu/gigas/data/p.generosa/Manilla_Pgenenerosa_blastx.tab)
- Pgenerosa Against Merceneria DB [Mercenaria_db_Pgen](https://gannet.fish.washington.edu/gigas/data/p.generosa/Pgenerosa_Mercenaria_db_blastx.tab)
- Pgenerosa Against Manilla DB [Manilla_db_Pgen](https://gannet.fish.washington.edu/gigas/data/p.generosa/Pgenerosa_Manilla_db_blastx.tab)

# Compairative Transcriptome Research

Northern Quahog (Mercenaria mercenaria) 
Transcriptome, yes! 

Genome assembly MADL_Memer_1
https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_021730395.1/

Paper 1 doi: 10.1186/s12864-021-08262-1

in this context, the characterization of the hard clam genome can provide highly needed resources to enable basic (e.g., oncogenesis and cancer transmission, adaptation biology) and applied (clam stock enhancement, genomic selection) sciences.

Using a combination of long and short-read sequencing technologies, a 1.86 Gb chromosome-level assembly of the clam genome was generated. The assembly was scaffolded into 19 chromosomes, with an N50 of 83 Mb. Genome annotation yielded 34,728 predicted protein-coding genes, markedly more than the few other members of the Venerida sequenced so far, with coding regions representing only 2% of the assembly. Indeed, more than half of the genome is composed of repeated elements, including transposable elements. Major chromosome rearrangements were detected between this assembly and another recent assembly derived from a genetically segregated clam stock. Comparative analysis of the clam genome allowed the identification of a marked diversification in immune-related proteins, particularly extensive tandem duplications and expansions in tumor necrosis factors (TNFs) and C1q domain-containing proteins, some of which were previously shown to play a role in clam interactions with infectious microbes. The study also generated a comparative repertoire highlighting the diversity and, in some instances, the specificity of LTR-retrotransposons elements, particularly Steamer elements in bivalves.

Genome Assembly ASM1480567v1.1
https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_014805675.1/
Transcriptome, yes! 

Paper 1 doi: 10.1186/s12915-020-00943-9

Here, we present a chromosome-level genome assembly of an economically significant bivalve, the hard clam Mercenaria mercenaria…Comparative genome analysis reveals that this massive expansion is characteristic of bivalves more generally. Reconstruction of the evolutionary history of molluscan IAP genes indicates that most originated in early metazoans and greatly expanded in Bivalvia through both lineage-specific tandem duplication and retroposition, with 37.1% of hard clam IAPs located on a single chromosome.




Manilla Clam
Ruditapes philippinarum (Manila clam)

https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_026571515.1/

Paper 1 doi: 10.1093/gbe/evac171 

We present the first long-read-based de novo genome assembly of a Manila clam, and a RNA-Seq multi-tissue analysis of 15 females and 15 males. The highly contiguous genome assembly was used as reference to investigate gene expression, alternative splicing, sequence evolution, tissue-specific co-expression networks, and sexual contrasting SNPs.




https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_009026015.1/

Paper 1 doi: 10.1038/s41598-021-86727-9

In the present study, we compared high-quality, small-RNA transcriptomes in three strains of the Manila clam Ruditapes philippinarum with specific shell-color patterns, artificially selected for six generations. Totals of 114 known and 208 novel miRNAs were identified by high-throughput sequencing, of which nine known and one novel miRNA were verified by stem-loop quantitative real time-polymerase chain reaction.

Paper 2 doi: 10.1016/j.isci.2019.08.049

In this study, we sequenced the whole genome of the Manila clam and investigated the molecular basis of its adaptation to hypoxia, acidification, and parasite stress with transcriptome sequencing and an RNA sequence analysis of different tissues and developmental stages to clarify these major issues.

Linked articles: The Whole-Genome and Transcriptome of the Manila Clam (Ruditapes philippinarum) Mun et al 2017 Genome Biology and Evolution
https://doi.org/10.1093/gbe/evx096

"In this current study, we sequenced the whole-genome
and transcriptome (three tissues) of R. philippinarum, using the Illumina HiSeq 2500 platform."

"Consequently, to improve accuracy of gene prediction, we incorporated R. philippinarum transcriptome data for annotating gene sets in the assembled genome. Our study provides the basic knowledge for understanding genomic features of R. philippinarum and our data will be useful for further comparative, systematic, and functional genomics studies in bivalve species."

"Furthermore, we performed transcriptome analysis with three different tissues in order to support genome annotation and then identified 41,275 transcripts which were annotated"

"Recently, transcriptome analysis of R. philippinarum using 454 pyrose- quencing technology revealed putative members of immuno- logical defense genes involved in complement cascades, apoptosis, and the toll like signaling pathways (Moreira et al. 2012)."

--Trinity assembly 




Archivesica marissinica
Chemosymbiotic clam
https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_014843695.1/

Paper 1: doi 10.1093/molbev/msaa241
"Here, we sequenced the genomes of the clam Archivesica marissinica (Bivalvia: Vesicomyidae) and its bacterial symbiont to understand the genomic/metabolic integration behind this symbiosis… Overall, the host-symbiont genomes show not only tight metabolic complementarity but also distinct signatures of coevolution allowing the vesicomyids to thrive in chemosynthesis-based ecosystems"

Paper 2: doi 10.1186/s12915-023-01551-z
"Conclusions: Thus, bivalves employ different pathways to adapt to the long-term co-existence with their bacterial symbionts, further highlighting the contribution of stochastic evolution to the independent gain of a symbiotic lifestyle in the lineage."



Pearl Oyster: 
"The completion of the pearl oyster genome sequence has accelerated studies on the genomes of other members of the bivalve family (Takeuchi et al. 2012, 2016; Zhang et al. 2012)." 


![image](https://user-images.githubusercontent.com/81712104/231642949-fd915044-48cb-465a-b705-ae2893c7a8ae.png)
